src/com/trainee/phani/PhaniTestClass.java

```diff
@@ -2,21 +2,26 @@
 import java.util.Scanner;
 public class PhaniTestClass {
 
-        public static void main(String []args){
-            Scanner sc=new Scanner(System.in);
-            int sum=0;
+        public static void main(String []args) {
+            Scanner sc = new Scanner(System.in);
+            int sum = 0;
             System.out.println("enter n value");
-            int n=sc.nextInt();
-            int array[]=new int[n];
-            System.out.println("enter the values");
-            for(int i=0;i<n;i++){
-                array[i]=sc.nextInt();
-            }
-            for(int j=0;j<n;j++){
+            try {
+                int n = sc.nextInt();
+                int array[] = new int[n];
+                System.out.println("enter the values");
+                for (int i = 0; i < n; i++) {
+                    array[i] = sc.nextInt();
+                }
+                for (int j = 0; j < n; j++) {
 
-                sum=sum+array[j];
+                    sum = sum + array[j];
+                }
+                System.out.println("Sum of the array is:" + sum);
+            }
+            catch(Exception e){
```

Comment:
Exception is the Super class.
The Best Practice is catching the Exact Exception